examples/keyboard.py

```diff
@@ -15,8 +15,8 @@
         elif gui.event.key == ti.GUI.WHEEL:
             x, y = gui.event.pos
             dt = gui.event.delta
```

Comment:
I have mixed feelings about naming this as dt. Normally, dt can be everything else but delta. Could you give it another name here?